config/crd/compute.gcp.crossplane.io_subnetworks.yaml

```diff
@@ -381,6 +385,9 @@ spec:
                       instance. The name must be 1-63 characters long, and comply
                       with RFC1035. The name must be unique within the subnetwork.'
                     type: string
+                required:
+                - ipCidrRange
+                - rangeName
```

Comment:
I was adding `+optional` tags to these fields and realised that they're not actually optional so I removed the `omitempty` tags.